GPSTest/build.gradle

```diff
@@ -6,6 +6,10 @@ apply plugin: 'dagger.hilt.android.plugin'
 android {
     compileSdkVersion 31
 
+    lint {
```

Comment:
I believe you should be able to get rid of this now that you updated the dependencies?

Reply:
> I believe you should be able to get rid of this now that you updated the dependencies?

Sure, It works.